usr/lib/systemd/system/proxysql.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=High Performance Advanced Proxy for MySQL
+
+[Service]
+Type=forking
+PIDFile=/var/lib/proxysql/proxysql.pid
+ExecStart=/usr/local/bin/proxysql
```

Comment:
Yes, you can put default parameters there, so I guess that the line will be :

ExecStart=/usr/local/bin/proxysql -c /etc/proxysql.cnf -d /var/lib/proxysql